### ISSUE TITLE
Calling issue with Microsoft Teams application

### ISSUE THREAD

**Issue by gs9339** [Umang96/munch_pe_custom_gsi_releases#1]
I've tried the Latest GSI and the one previous GSI. Had clean installed both of them. In both the versions their is problem with teams call, the voice from teams is extremely noisy and you can't understand anything. Mic works during team call.

**Comment by Umang96** [comment]
You are saing that you can't hear other people and other people can hear you, correct?
Also, you using earphone? If yes then which one?

**Comment by gs9339** [comment]
Yes I can't hear other people clearly. Nope, not using any earphones.

**Comment by Umang96** [comment]
I tested with a teams meeting and found that volume is a bit too low. I had to listen carefully to understand. I will try to find a fix for this.
